docs/reference/api/pages.md

```diff
@@ -1,33 +1,37 @@
 # Page type REST endpoints
 
-All [page types](#TODO) use a single set of API endpoints, unlike piece types. Difference in page type will primarily influence returned document properties based on the page type configurations.
+All [page types](/reference/glossary.md#page) use a single set of API endpoints, unlike piece types. Difference in page type will primarily influence returned document properties based on the page type configurations.
```

Comment:
Might just be me, but the second sentence seems....needlessly complicated? Feels like we're saying something very simple with a lot of words.

Reply:
i can remove it. it's not doing a lot.